lib/middleware/isAdminSession.js

```diff
@@ -0,0 +1,91 @@
+//const config = require('../config');
+const logger = require('../logger');
+const licenceListSort = require('../licenceListSort');
+//const cookieSession = require('cookie-session');
+const accesslicenceService = require('../services/accesslicenceService');
```

Comment:
This should be `require('../services/accessLicenceService');`, with a capital `L`